src/journal/journalctl.c

```diff
@@ -330,6 +330,7 @@ static int parse_lines(const char *arg, bool graceful) {
 
 default_noarg:
         arg_lines = 10;
```

Comment:
Should be ARG_LINES_DEFAULT ?

Reply:
No, this is applied when `-n` is specified without argument. `ARG_LINES_DEFAULT` is used to indicate `-n` is not specified at all, which is what we initialize `arg_lines` to.

Reply:
Ah, I see. Thanks. I thought `ARG_LINES_DEFAULT` is `10`, but it is not. Sorry for the noise.